templates/RA.html.tmpl

```diff
@@ -423,8 +422,9 @@
 				{{#each data.sections.sec5.risks }}
 				<tr>
 					<td class="lean">{{key}}</td>
-					<td class="lean">{{{requirementsKey}}}</td>
-					<td class="content-wrappable">{{{requirements}}}</td>
+                    {{#each requirements}}
```

Comment:
All the requirements in the list must appear in the same column. So, the each loop should be inside the td.